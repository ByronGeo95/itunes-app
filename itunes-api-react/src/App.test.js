//Created by: Byron Georgopoulos
//Created on: 11/09/2020
//Last Updated on: 11/09/2020
//Created for: HyperionDev - L02T21: Capstone II (Full-Stack Web-Development Bootcamp)
//Description: This is the React Snapshot Test of my Full-Stack Express & React Web App that uses the iTunes API.

import React from 'react';
import renderer from 'react-test-renderer';

test('H1 Renders Correctly', () => {
  const tree = renderer
    .create(<h1>iTunes Store API:</h1>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
