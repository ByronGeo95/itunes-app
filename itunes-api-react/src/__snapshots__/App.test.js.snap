// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 Renders Correctly 1`] = `
<h1>
  iTunes Store API:
</h1>
`;

exports[`Renders Correctly 1`] = `null`;
